docker_entrypoint.sh

```diff
@@ -18,4 +18,8 @@ else
   printf "\n${help_result}"
 fi
 
+if [[ -n "${ENABLE_HEAPTRACK}" ]]; then
```

Comment:
The idea here is to only run heaptrack if the string value for the env var is NOT null